hybrasyl/Objects/User.cs

```diff
@@ -2327,7 +2327,7 @@ public void ShowLearnSkillMenu(Merchant merchant)
             var merchantSkills = new MerchantSkills();
             merchantSkills.Skills = new List<MerchantSkill>();
 
-            foreach (var skill in merchant.Roles.Train.Where(x => x.Type == "Skill").OrderBy(y => y.Name))
+            foreach (var skill in merchant.Roles.Train.Where(x => x.Type == "Skill" && x.Class.Contains(Class)).OrderBy(y => y.Name))

```

Comment:
Is this right? I think this needs to be
`(x.Class.Contains(Class) || x.Class.Contains(Class.Peasant))` otherwise you're only getting class-specific skills and spells.

Reply:
hmmm